services/getUsers.js

```diff
@@ -0,0 +1,37 @@
+const {ddb, ddbClient} = require('../dbClient/ddbClient');
+const { TABLE_NAME } = require('../shared/constants');
+
+const queryAllUsersByCustomerId = (customerId) => {
+    const params = {
+        TableName: TABLE_NAME,
+        ExpressionAttributeValues: {
+            ':customerId': `CUSTOMER_ID::${customerId}`
+        },
+        KeyConditionExpression: 'pk = :customerId'
+    };
+
+    return ddbClient.query(params).promise();
+}
+
+const getUserByCustomerIdAndEmail = (customerId, email) => {
+    const params = {
+        TableName: TABLE_NAME,
+        Key: {
+            pk: `CUSTOMER_ID::${customerId}`,
+            sk: `EMAIL::${email}`
+        }
+    };
+
+    return ddbClient.get(params).promise();
+}
+
+const scanUserTable = () => {
+    const params = {
+        TableName: TABLE_NAME
+    };
+
+    return ddbClient.scan(params).promise();
```

Comment:
A single Scan operation reads up to the maximum number of items set (if using the Limit parameter) or a maximum of 1 MB

Similar to `query` API the scan also has a limitation : https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html